models/nlp/classification/huggingface/inference.py

```diff
@@ -0,0 +1,56 @@
+import zipfile
+
+import torch
+import valohai
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+from utils.serializers import get_serializer
+
+
+def load_model(model_path):
+    tokenizer = AutoTokenizer.from_pretrained(model_path)
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    model = AutoModelForSequenceClassification.from_pretrained(model_path)
+    model.to(device)
+    return tokenizer, model, device
+
+
+def predict(text, tokenizer, model, device):
```

Comment:
Would (some) type hints be possible? (Repeated for untyped functions everywhere.)